scripts/locale-kit.ts

```diff
@@ -41,22 +41,41 @@ function getUsedKeys(content: string) {
         }
         return undefined
     }
-    const addKey = (node: ts.Node) => {
-        if (ts.isStringLiteralLike(node)) {
-            keys.add(node.text)
-        }
-    }
     const transformer = (context: ts.TransformationContext) => (rootNode: ts.Node) => {
+        const setFromVariableWrapper = (variableValue: string): ((node: ts.Node) => ts.Node) => {
+            const setFromVariable = (node: ts.Node): ts.Node => {
+                if (
+                    ts.isVariableDeclaration(node) &&
```

Comment:
Please check if the variable declaration is a `const` declaration.

Reply:
```ts
if (ts.isVariableDeclarationList(node.parent)) {
    const constFlag = node.parent.flags & ts.NodeFlags.Const
}
```